.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: CI build of the standard
+
+on: [push, pull_request]
```

Comment:
I included the push trigger on the assumption that it would trigger CI on a users forked copy of the repo - that way they can see if the code passes CI before they create the PR. I haven't tested that out yet though!

Reply:
Change made as suggested